src/components/Header/Header.js

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
-import { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import BnrHead from './BnrHead/BnrHead';
 import WrapHead from './WrapHead/WrapHead';
 import GroupHead from './GroupHead/GroupHead';
-import styled from 'styled-components';
 
 const Header = () => {
   const [isFixed, setIsFixed] = useState(false);
+  const navigator = useNavigate();
+  const [isOpenModal, setIsOpenModal] = useState(false);
+  const userToken = localStorage.getItem('token');
+
+  const handleModalConfirm = result => {
+    const btnText = result.target.textContent;
+    if (btnText === '확인') {
+      localStorage.removeItem('token');
+      localStorage.removeItem('nickname');
+      setIsOpenModal(prev => !prev);
+    } else if (btnText === '취소') {
+      setIsOpenModal(prev => !prev);
+    }
```

Comment:
```suggestion
    } 
     setIsOpenModal(prev => !prev);
```

동일한 동작을 하는 거라면 이와 같이 구현할 수 있겠습니다.

Reply:
수정완료하였습니다🫡